root/static/styles/layout.less

```diff
@@ -386,6 +386,10 @@ div.warning img.warning {
     margin-top: 0;
 }
 
+#sidebar ul.external_links {
```

Comment:
We have a block for `#sidebar ul.external_links` right below this on line 398, so it'd be good to put this rule inside the existing block.